spec/features/01_visitor_sees_fruit_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+feature "visitor sees content" do
+  scenario "fruits are on the page" do
```

Comment:
for this `feature` block, you can be much more explicit. `feature "when visiting the fruits index page" do`

Reply:
added more depth